Doc/library/sys.rst

```diff
@@ -1034,7 +1034,13 @@ always available.
 
    .. attribute:: hash_info.nan
 
-      (This attribute is no longer used)
+      The hash value returned for not-a-number (NaN).
+
+      This hash value is only used by the :c:func:`Py_HashDouble` C function if
+      the argument is not-a-number (NaN).
```

Comment:
Why is this sentence useful? Is it accurate?

Reply:
I rephrased the Py_HashDouble() doc to no longer mention NaN directly. So I reverted these sys.rst changes.